calico/_includes/release-notes/v3.23.0-release-notes.md

```diff
@@ -0,0 +1,145 @@
+ - epbf: Fixed in bug in ICMP connection tracking [calico #5792](https://github.com/projectcalico/calico/pull/5792) (@tomastigera)
+ - ebpf: Fixed reusing stale conntrack TCP entries. [calico #5804](https://github.com/projectcalico/calico/pull/5804) (@tomastigera)
+ - ebpf: Fix that policy would see 0.0.0.0:0 as the post-nat destination for SYN packets that had a conntrack hit. [calico #5768](https://github.com/projectcalico/calico/pull/5768) (@tomastigera)
+ - ebpf: Fixed unconnected udp affinity - avoids be selection for each packet. [calico #5617](https://github.com/projectcalico/calico/pull/5617) (@tomastigera)
+
+##### Windows
+
+ - Fix Calico for Windows install script not enabling WinDSR on EKS [calico #5903](https://github.com/projectcalico/calico/pull/5903) (@lmm)
+ - Fix invalid parameter to prevented DSR enablement from being passed to kube-proxy in Windows. [calico #5710](https://github.com/projectcalico/calico/pull/5710) (@lmm)
+ - Fix windows kube-proxy config incompatible with k8s 1.23 [calico #5653](https://github.com/projectcalico/calico/pull/5653) (@lmm)
+
+
+#### Other changes
+
+##### General
+
+ - Operator monitors BGP configuration resource to trigger rolling updates as needed [calico #6017](https://github.com/projectcalico/calico/pull/6017) (@caseydavenport)
+ - go version update to 1.17.9 [calico #6002](https://github.com/projectcalico/calico/pull/6002) (@doublek)
+ - Attach SHA256SUMS as part of release, including checksums for all release artifacts. [calico #5964](https://github.com/projectcalico/calico/pull/5964) (@caseydavenport)
+ - Update to CNI plugins v1.1.1 [calico #5944](https://github.com/projectcalico/calico/pull/5944) (@caseydavenport)
+ - Helm chart allows configuration of tigera-operator resource requirements. [calico #5934](https://github.com/projectcalico/calico/pull/5934) (@sriramy)
+ - Add defensive checks to CNI plugin installer to improve reliability. [calico #5905](https://github.com/projectcalico/calico/pull/5905) (@caseydavenport)
+ - Increase Typha's "max fall behind" timeout to 300s.  This means that Typha will keep up to 300s of history if needed instead of 90s (thus increasing RAM usage if clients are falling a bit behind) but it reduces the chance of disconnecting a client spuriously in a very large cluster. [calico #5885](https://github.com/projectcalico/calico/pull/5885) (@fasaxc)
+ - Add Felix configuration option for programming floating IPs. [calico #5861](https://github.com/projectcalico/calico/pull/5861) (@caseydavenport)
+ - Produce both static and dynamic builds of the Calico CNI plugin, fixes compatibility with systems that do not include glibc. [calico #5826](https://github.com/projectcalico/calico/pull/5826) (@caseydavenport)
+ - Build components with go 1.17.8 to pick up CVE fixes. [calico #5817](https://github.com/projectcalico/calico/pull/5817) (@doublek)
```

Comment:
This was superceded by the 1.17.9 bullet point above:

>  - go version update to 1.17.9 [calico #6002](https://github.com/projectcalico/calico/pull/6002) (@doublek)


We can just remove this one.